Dockerfile

```diff
@@ -6,15 +6,15 @@ MAINTAINER zsx <thinkernel@gmail.com>
 ENV GERRIT_HOME /var/gerrit
 ENV GERRIT_SITE ${GERRIT_HOME}/review_site
 ENV GERRIT_WAR ${GERRIT_HOME}/gerrit.war
-ENV GERRIT_VERSION 2.15.2
+ENV GERRIT_VERSION 2.15.3
 ENV GERRIT_USER gerrit2
 ENV GERRIT_INIT_ARGS ""
 
 # Add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
 RUN adduser -D -h "${GERRIT_HOME}" -g "Gerrit User" -s /sbin/nologin "${GERRIT_USER}"
 
 RUN set -x \
-    && apk add --update --no-cache git openssh openssl bash perl perl-cgi git-gitweb curl su-exec procmail
+    && apk add --update --no-cache git openssh-client openssl bash perl perl-cgi git-gitweb curl su-exec procmail
```

Comment:
Are you sure openssh-client is good enough? I added this years ago. I can't remember why I used openssh instead of openssh-client. :P So I hope you've already test this on your environment.

Reply:
`openssh` is a group, it installs both `openssh-client` and `openssh-server` and I figured Gerrit doesn't need the `server`. I have tested it, yes and everything works as expected:
```
2018-07-25 10:54:52,600 +0000] 053a33df test_user a/1000000 LOGIN FROM 172.1.19.78
[2018-07-25 10:54:53,313 +0000] 053a33df test_user a/1000000 git-receive-pack./dm 72ms 487ms 0
[2018-07-25 10:54:53,340 +0000] 053a33df test_user a/1000000 LOGOUT
[2018-07-25 10:54:57,433 +0000] 65668f08 test_user a/1000000 LOGIN FROM 172.1.19.78
[2018-07-25 10:54:57,627 +0000] 65668f08 test_user a/1000000 git-upload-pack./dm 7ms 88ms 0
[2018-07-25 10:54:57,653 +0000] 65668f08 test_user a/1000000 LOGOUT
[2018-07-25 10:54:59,684 +0000] e5511f21 test_user a/1000000 LOGIN FROM 172.1.19.78
[2018-07-25 10:54:59,882 +0000] e5511f21 test_user a/1000000 git-receive-pack./dm 3ms 89ms 0
[2018-07-25 10:54:59,953 +0000] e5511f21 test_user a/1000000 LOGOUT
```